plugin/trino-delta-lake/src/test/resources/databricks/uri/_delta_log/00000000000000000005.json

```diff
@@ -0,0 +1,2 @@
+{"add":{"path":"part=a%2525percent/part-00000-f6243f45-fea9-4d0e-8744-ef23cfe2b99c.c000.snappy.parquet","partitionValues":{"part":"a%percent"},"size":475,"modificationTime":1671166390000,"dataChange":true,"stats":"{\"numRecords\":1,\"minValues\":{\"y\":5},\"maxValues\":{\"y\":5},\"nullCount\":{\"y\":0}}"}}
```

Comment:
why `a%2525percent` and not `a%25percent`?

note also that the resource file path is `part=a%25percent/part-00000-f6243f45-fea9-4d0e-8744-ef23cfe2b99c.c000.snappy.parquet` while it probably should be `part=a%percent/part-00000-f6243f45-fea9-4d0e-8744-ef23cfe2b99c.c000.snappy.parquet`

Reply:
The path is escaped at `DeltaLakePageSink#makePartName` and encoded at `DeltaLakeMetadata#toUriFormat`. 

> note also that the resource file path is part=a%25percent/part-00000-f6243f45-fea9-4d0e-8744-ef23cfe2b99c.c000.snappy.parquet while it probably should be part=a%percent/part-00000-f6243f45-fea9-4d0e-8744-ef23cfe2b99c.c000.snappy.parquet

It's exactly weird, but the resource path is the same as what Spark generated. 

Filed #15499 to investigate the details.